app/v2/models/responses/MessageResponseWithoutBody.scala

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v2.models.responses
+
+import play.api.libs.json.Json
+import v2.models.MessageId
+import v2.models.request.MessageType
+
+import java.time.OffsetDateTime
+
+object MessageResponseWithoutBody {
+  implicit lazy val messageResponseFormatWithoutBody = Json.format[MessageResponseWithoutBody]
+}
+
+case class MessageResponseWithoutBody(
+  id: MessageId,
+  received: OffsetDateTime,
+  messageType: MessageType,
+  body: Option[String] = None
```

Comment:
The name MessageResponseWithoutBody doesn't make sense as it does contain the body

Reply:
You do have a point :grinning:

Will address.